tests/acceptance/bootstrap/FeatureContext.php

```diff
@@ -165,9 +165,16 @@ public function theSprintDoesNotExist($title)
 		}
 	}
 
-	private function getPhabricatorProjectFromTitle($title)
+	private function getOrCreatePhabricatorProjectFromTitle($title)
 	{
-		return App::make('phabricator')->queryProjectByTitle($title);
+		$phabricator =  App::make('phabricator');
+		$project = $phabricator->queryProjectByTitle($title);
+
+		if (!$project) {
```

Comment:
Gah, different code styles everywhere. Confusing!

Indeed your snippet is even shorter. I will use it :)